app/(auth)/onboarding.tsx

```diff
@@ -0,0 +1,247 @@
+import Screen from '@src/components/screen';
+import Assets from '@src/theme/assets';
+import screen from '@src/utils/helpers';
+import { Link } from 'expo-router';
+import React, { useEffect, useRef, useState } from 'react';
+import { Animated, FlatList, View } from 'react-native';
+import { Button, XStack, Text, Stack, YStack } from 'tamagui';
+
+interface InformationCarouselContent {
+    title: string;
+    description: string;
+}
+
+const imageCarouselData = [
+    { key: 'spacer-1' },
+    {
+        img: Assets.Girl1(),
+        key: 'item-1',
+    },
+    {
+        img: Assets.Girl2(),
+        key: 'item-2',
+    },
+    {
+        img: Assets.Girl3(),
+        key: 'item-3',
+    },
+    {
+        img: Assets.Girl1(),
+        key: 'item-4',
+    },
+    {
+        img: Assets.Girl2(),
+        key: 'item-5',
+    },
+    {
+        img: Assets.Girl3(),
+        key: 'item-6',
+    },
+    { key: 'spacer-2' },
+];
+
+const duplicateDataWithUniqueKeys = () => {
```

Comment:
Why do we duplicate the data? Didn't really understand this

Reply:
for the ilussion of infinite looping so that the last image has the first image an its right